configs/samples/pjsip.conf.sample

```diff
@@ -137,8 +137,8 @@
 ;protocol=udp
 ;bind=0.0.0.0
 ;local_net=192.0.2.0/24
```

Comment:
Bad private CIDR block.  It should be 192.168.2.0/24.

This is outside the changed lines so I wonder what'll happen.